### DISCUSSION TITLE
Unknown flag --format

### DISCUSSION THREAD

**Discussion by bbodensieck** [cerbos/cerbos#1615] [Q&A] [answered]
Hey, 
in the [docs](https://docs.cerbos.dev/cerbos/latest/cli/cerbosctl.html#get) is an example for getting policies from the storage.
```
cerbosctl get derived_roles my_derived_roles --format=yaml
```
I am getting an error `cerbosctl: error: unknown flag --format`. I guess the correct flag is `output` instead of `format` so the command would look like this:
```
cerbosctl get derived_roles my_derived_roles --output=yaml
```
Can you confirm?

Cheers
Björn

**Comment by oguzhand95** [comment, marked as answer]
Hey Björn,

Yes, the correct flag is `--output`.

Thanks for reporting!

oguzhan
